pkg/provider/metric_store.go

```diff
@@ -140,7 +140,7 @@ func (s *MetricStore) insertCustomMetric(value custom_metrics.MetricValue) {
 		labelsKey = hashLabelMap(selector.MatchLabels)
 	}
 
-	metric := metricName(value.Metric.Name)
+	metric := metricName(strings.ToLower(value.Metric.Name))
```

Comment:
Was there a particular reason to introduce this change? Asking out of curiousity.

Reply:
This was intended for another bug fix unrelated to this, I accidentally commit this.

Removed again.